### ISSUE TITLE
Gescante Seiten werden nicht erkannt

### ISSUE THREAD

**Issue by GRRRR11** [DatabayAG/ScanAssessment#61]
Hallo Zusammen,

ich habe bei dem in der letzten Meldung beschriebenen Test jetzt alles durchgescant und analysiert. Dabei werden zwei Seiten nicht erkannt. Bei 568 gescanten und analysierten Seiten ist das ein hervorragender Schnitt. Allerdings ist es ärgerlich, dass ich es bisher nicht geschafft habe, diese beiden Seiten ins System zu bekommen. Ursprünglich waren es 4 Seiten (alle von unterschiedlichen Klausurbögen). Bei zwei Seiten hat ein neuer Scan abhilfe geschaffen, bei den beiden übrigen hat bisher nichts geholfen. Keine höhere Auflösung beim Scannen, kein dunkler oder heller scannen etc. Wäre in einer echten Klausur-Situation fatal, da es ja auch keine Möglichkeit gibt, die fehlenden Antworten der Studierenden ins System zu "tippen".
Äußerlich gibt es bei den beiden Seiten keinen Unterschied zu den anderen. Die QR-Codes sind sauber, fransen nicht aus oder ähnliches.
Nach der Analyse sind die Seiten aus "Noch nicht analysierte Dateien" verschwunden, werden aber nicht den restlichen Seiten des Klausurbogens zugeordnet. Stattdessen taucht unter "Analysierte Dateien" etwas wie das hier auf: "analysed/769/doc01524120171116131438_002-0.gif"
In so fern komisch, da die Seite dem Bogen 762 zugeordnet werden müsste. Irgendwann verschwindet die Datei dann auch einfach.

**Comment by gvollbach** [comment]
Auch hier bräuchte ich dann mal den entsprechenden Log und am besten die entsprechenden Scans. Wenn die Seite verschwindet spricht das dafür, das die einem anderen Test zugeordnet wurde, die werden dann automatisch verschoben in den entsprechenden Test Ordner wo sie eigentlich hingehören.

**Comment by GRRRR11** [comment]
Ich habs mal überprüft. die gescannten Seiten tauchen nirgendwo mehr auf.

**Comment by gvollbach** [comment]
Problem mit 762_4 ist halt:
[8gfdt] [2017-11-15 17:32:27.583695] .ERROR:  QR Code could not be read! 

Kannst du bei den nicht funktionierenden bitte einmal versuchen die statt in Schwarz/Weiß in Farbe einzuscannen?

**Comment by GRRRR11** [comment]
Ich habe jetzt zuletzt nur die fehlenden Seiten einzeln gescannt, mit einem Flachbettscanner, in Farbe, mit 600dpi und höchster Qualität des erstellten TIFF. Leider ohne Erfolg. Allerdings ist eine Fehlerquote von 0,35% bei einem ersten Test schon mal der absolute Hammer!
Von der Auswertung einer Klausur her ist das alles kein Problem, nach der Benutzerzuweisung und Rückführung kann man die fehlenden Daten über die manuelle Bewertung nachtragen und wen der/die Studierende Einsicht verlangt ist das Original noch da. Umständlich wirds ein wenig, wenn die erste Seite mit der Matrikel-Matrix nicht erkannt wird, ist aber alles machbar.

**Comment by gvollbach** [comment]
Ja, ich bin auch recht positiv überrascht wie gering der Fehler ist, ich denke das beste wäre, noch sowas wie eine Einzel Zuordnung einzuführen, sprich wenn die Dokumenten Id nicht automatisch aus dem QR Code gelesen werden kann, kann ein Dozent dann manuell die Id nachtragen, aber das wäre auch dann ein Feature, wenn es für dich ok ist, würde ich das ganze hier als Feature taggen, dann geht es nicht unter.

Stimmt das mit der manuellen Bewertung wollt ich dich gestern noch fragen, ist zwar auch etwas umständlich aber es geht zumindest und wenn es bei knapp 600 Seiten nur 2 betrifft, kann man glaub ich damit leben.

Übrigens danke schon mal das du hier mal ein Test in größerem Ausmaß gemacht hast. Könntest du mir beizeiten mal ein paar Eckdaten zu deinem Test zukommen lassen, sprich wie lange hat das jetzt alles gedauert, wie groß ist das Auswertungsverzeichnis, usw.

**Comment by GRRRR11** [comment]
Ich muss dazu sowieso eine Doku schreiben, die kann ich dir dann zukommen lassen. Bisher hat alles mit Einscannen, Analyse, Revision, und Benutzerzuweisung etwa 9 Stunden gedauert, bei 142 Klausurbögen á 4 Seiten mit einem "Standard"-Einzugsscanner (Multifunktionsdrucker Kyocera TASKalfa 3551ci).

**Comment by gvollbach** [comment]
Das wäre natürlich optimal, dank dir schon mal.
